Spring-Boot/Spring-Boot-Academy/src/test/java/tw/waterball/judgegirl/springboot/academy/controllers/HomeworkControllerTest.java

```diff
@@ -99,6 +102,27 @@ public void GivenOneHomeworkCreated_WhenGetHomeworkById_ShouldRespondHomework()
         assertEquals(homework, actualHomework);
     }
 
+    @Test
+    public void GivenOneHomeworkCreated_WhenDeleteHomeworkById_ShouldSucceed() throws Exception {
+        var homework = createHomeworkAndGet(HOMEWORK_NAME);
+
+        deleteHomework(homework.id).andExpect(status().isOk());
+
+        assertFalse(homeworkRepository.findHomeworkById(homework.id).isPresent());
+    }
+
+    @Test
+    public void GivenManyHomeworkCreated_WhenGetAllHomework_ShouldReturnAllHomework() throws Exception {
+        List<HomeworkView> homeworkList = createHomeworkAndGet(3);
+
+        List<HomeworkView> actualHomeworkList = getBody(getAllHomework(), new TypeReference<>() {
+        });
```

Comment:
use `var` instead of the concrete type `List<HomeworkView>` since the types are possibly changed